### DISCUSSION TITLE
Reduce risks, lower barriers, allow more enthusiasts to participate, and increase project activity index

### DISCUSSION THREAD

**Discussion by jekc2008** [filecoin-project/FIPs#1088] [Enhancements - Technical]
Setting a lower limit for computing power below 100TB or 10TB exempts punishment, which is equivalent to providing entry-level benefits to enthusiasts and allowing more SPs to participate in the Filecoin project.

设定算力下限，低于100TB，或者低于10TB的算力，免除惩罚，相当于给爱好者，提供入门福利，让更多的SP参与filecoin项目。

**Comment by rvagg** [comment]
Converted this to a discussion. It's not the first time I've heard a proposal related to decreasing the barrier-to-entry and I agree that it would be very healthy to lower it right down to allow people to at least dip their toes into being an SP and make the learning curve less steep. Ultimately Filecoin needs fairly serious storage providers with serious hardware and networks and dedication to providing a storage service, but it would be unfortunate if you have to come to Filecoin with all of that in place in order to just get started.

Perhaps we can use this discussion topic to raise other proposals or link to existing discussions about this topic?
